src/Business Foundation/App/NoSeries/src/Single/NoSeriesImplementation.Enum.al

```diff
@@ -0,0 +1,24 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+enum 397 "No. Series Implementation" implements "No. Series - Single"
+{
+    Access = Public;
+    Extensible = true;
```

Comment:
Given how this enum is used in NoSeriesImpl.Codeunit.al:

```al
    local procedure GetImplementation(var NoSeriesLine: Record "No. Series Line"): Interface "No. Series - Single"
    begin
        if NoSeriesLine."Allow Gaps in Nos." then // TODO: Enum needs to be specified on the table and retrieved from there
            exit(Enum::"No. Series Implementation"::Sequence);
        exit(Enum::"No. Series Implementation"::Normal);
    end;
```

... I am not sure that this should really be extensible.

Reply:
Agreed. This can only be set to extensible, once the enum is implemented in the table and on the page.

Reply:
We are now fetching the implementation based on the enum set on the table. See NoSeries.Codeunit.al function GetImplementation.